deploy/ngsadev/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "zone": "dev",
+    "region": "dev",
```

Comment:
I don't think it's necessary to use the full names with "." - "name" and "domain" should be fine.

Am I missing something?

Reply:
I used this naming to be consistent with the environment variable names (e.g $ASB_DOMAIN and $ASB_NGSA_MI_NAME) from the README. It is not documented yet but I will be adding instructions to generate a .json using these environment variables.